### ISSUE TITLE
Book preview screen reader accessibility

### ISSUE THREAD

**Issue by gestchild** [wellcomecollection/wellcomecollection.org#4188]
## Who it's for

People who use screen readers to interact with our website

## Problem to be solved

The book preview section on work pages is intended to allow people to gain an overview of the content of a book. However, it relies on viewing images and there is currently no useful alt text available in place of the images. 

## User Story/Job

In order to gain an understanding of the content of a book, from the preview section of the work page,
As a person who relies on a screen reader
I need appropriate alt text for the images displayed

## Acceptance Criteria
- [ ] A screen reader should be able to read out appropriate alt text for the images in the work preview
